modules/sor/sorV2/lib/utils/helpers.ts

```diff
@@ -120,7 +120,7 @@ export function getHookState(pool: PrismaPoolAndHookWithDynamic): HookState | un
             };
         }
         default:
-            if (hookData.type) {
+            if (hookData.type && hookData.type !== 'MEV_TAX') {
```

Comment:
can elaborate this change?

Reply:
is mev tax going to be implemented? i assumed it's invisible to sor

Reply:
Oh, thats right. Looks good!